components/table/ReportTable/utils.ts

```diff
@@ -152,10 +154,49 @@ export const handlePdfExportClick = (
 ) => {
     const JsPDF = jsPDF
     const doc = new JsPDF()
+    const startX = 14
 
     const groupedEntries = toArray(
         groupBy(timesheetEntries, (entry) => entry.timesheet.id)
     )
+
+    if (groupedEntries.length > 1) {
+        let totalHours = 0
+        const allEmployeeHours: EmployeeHours[] = []
```

Comment:
As a general rule of thumb: Whenever you write `let`, I would suggest reconsidering if it's possible to do the same thing with `const`. This will not always apply, but it's a good thing to keep in mind.